plaidml2/bridge/keras/backend_test.py

```diff
@@ -476,21 +477,39 @@ def testFlatten(self, b, x):
     def testEye(self, b, *args):
         return b.eye(*args)
 
-    @unittest.skip('TODO: convert to EDSL')
     def testTileIdentity(self):
-        x = pkb.variable(m(3))
-        op = tile.Operation('function (I[N]) -> (O) { O = I; }', [('I', x)],
-                            [('O', tile.Shape(x.shape.dtype, (3,)))])
-        output = op.sole_output().eval()
+        I = pkb.variable(m(3)).tensor
+        N = edsl.TensorDim()
+        I.bind_dims(N)
+        O = edsl.TensorOutput(N)
+        i = edsl.TensorIndex()
+        O[i] = I[i]
+        opId = pkb._KerasNode('tile_identity', name='opId', tensor=O)
+        pkb.eval(opId)
         return 0
 
-    @unittest.skip('TODO: convert to EDSL')
+    #@unittest.skip('TODO: convert to EDSL')
```

Comment:
Since two outputs isn't currently supported with the EDSL, maybe leave the skip in with a note to that effect